src/Chainweb/Pact/Service/Types.hs

```diff
@@ -46,6 +46,7 @@ data PactException
   = BlockValidationFailure Text
   | PactInternalError Text
   | NoBlockValidatedYet
+  | TransactionValidationException [(PactHash, Text)]
```

Comment:
We don't need to collect them all (I changed the routine to throw on first error) so let's make this

```
| TransactionValidationException !PactHash !Text
```